### PR TITLE
Feature/zig 2244

### PR DESCRIPTION
**Pull request by muradmalik777**

### CODE REVIEW

File: src/components/Forms/CopyTraderForm/CopyTraderForm.js [hunk 1]
```diff
@@ -257,6 +258,14 @@ const CopyTraderForm = ({ provider, onClose, onSuccess }) => {
     handleSubmit(onSubmit);
   };
 
+  const redirectToHelp = () => {
+    if (typeof window !== "undefined") {
```
Comment (cwagner22) [inline src/components/Forms/CopyTraderForm/CopyTraderForm.js:262]: This is not needed, gatsby static site rendering won't run this method.

Comment (muradmalik777) [reply src/components/Forms/CopyTraderForm/CopyTraderForm.js:262]: you sure, because previously it broke the build?

Comment (muradmalik777) [reply src/components/Forms/CopyTraderForm/CopyTraderForm.js:262]: can we merge this now and I will clean later, because i have to change in test as well

Comment (cwagner22) [reply src/components/Forms/CopyTraderForm/CopyTraderForm.js:262]: Only when the code is run at init time. Not inside the functions.

**Pull request merged**